protocol/messenger.go

```diff
@@ -1503,18 +1503,14 @@ func (m *Messenger) watchConnectionChange() {
 		// Instead we will poll the connection status.
 		m.logger.Warn("using WakuV1, can't watch connection changes, this might be have side-effects")
 		go pollConnectionStatus()
-		return
-	}
+	} else {
```

Comment:
I actually think `return` is better comparing to `else`. 
It's a check of `err != nil`, so returning makes sense there. If we will have to add more code with conditions after `go subscribedConnectionStatus(subscription)`, the tabulation will grow to the right.

Reply:
Agree, fixed.